### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by VedanthB**
- added header
- added footer
- added home page layout (cards and image slider)
- 😎 Browse the preview: [https://deploy-preview-1--romantic-mccarthy-7219f3.netlify.app/](https://deploy-preview-1--romantic-mccarthy-7219f3.netlify.app/)

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+@import url("https://fonts.googleapis.com/css2?family=Fruktur&display=swap");
+
+:root {
+  --regal-blue: #2a2840;
+  --regal-blue-dark: #000022;
+  --regal-blue-transparent: #2a284083;
+  --fruktur-font: "Fruktur", cursive;
```
Comment (ArjunGTX) [inline index.css:7]: Better to use a generic variable name for font so that in case you change the font it makes more sense

**Review by ArjunGTX — APPROVED** [review]
Everything else looks good

**Review by tewarimohit — APPROVED** [review]
Looks good to me to merge.

File: index.css (hunk 1 above)
Comment (VedanthB) [reply index.css:7]: okay

**Pull request merged**